tests/simpleTests.R

```diff
@@ -65,4 +65,32 @@ if (Sys.getenv("Run_RPushbullet_Tests")=="yes") {
     str(fromJSON(pbPost(type="file", url=descfile, recipients = RPushbullet:::.getNames()[1],
                         email = res$receiver_email)[[1]]))
 
+    ## Test hierarchy of arguments with channel specified:
+    ## 1) All three should send to recipients.
+    ## 2) Email and channel should send to email.
+    ## 3) Recipients and channel should send to recipients
+    ## 4) Only channel should send to channel.
+
+    ## Post a note with recipients, email and channel specified.
+    result<-fromJSON(pbPost(type="note","A Simple Test","We think this should work.\nWe really do.",recipients = RPushbullet:::.getNames()[1],email = "mike.birdgeneau@gmail.com",channel = "rpushbullet_test")[[1]])
+    if(is.null(result$target_device_iden)){
+	stop("Test Failed.")
+    }
+
+    ## Post a note with email and channel specified.
+    result<-fromJSON(pbPost(type="note","A Simple Test","We think this should work.\nWe really do.",email = "mike.birdgeneau@gmail.com",channel = "rpushbullet_test")[[1]])
+    if(is.null(result$receiver_email)){
+	stop("Test Failed.")
+    }
+
+    ## Post a note with recipients and channel specified.
+    result<-fromJSON(pbPost(type="note","A Simple Test","We think this should work.\nWe really do.",recipients = RPushbullet:::.getNames()[1],channel = "rpushbullet_test")[[1]])
+    if(is.null(result$target_device_iden)){
+	stop("Test Failed.")
+    }
+
+    ## Post a note with only the channel specified.
+    str(fromJSON(pbPost(type="note","A Simple Test","We think this should work.\nWe really do.",channel = "rpushbullet_test",verbose=TRUE)[[1]]))
```

Comment:
`rpushbullet_test` is a channel I created to test this package. This channel will only work when tested with my API key. 
Otherwise, this would have to be changed to a channel owned by a user performing the testing.